docs/3_development/2_tech-stack/Typescript/readme.md

```diff
@@ -120,7 +120,7 @@ jobs:
 ::: note This is a mostly automated process to enable consistency and make it hassle-free
 to release new versions more often
 :::
-::: caution Make sure the repository is using squash merging and branching rules as described in [Setup repository section](../../1_development-flow/1_setup_repository.md)!
+::: caution Make sure the repository is using squash merging and branching rules as described in [Setup repository section](/development/development-flow/setup_repository)!
```

Comment:
why the update of links? Targetting specific relative .md files provides better safety as it will break in build time if file is missing afaik. They get converted to your version after build

Reply:
It will break the build in case of broken links like this too. But from my perspective, it is better because it unties the current file position from the referencing document by converting relative to absolute paths. Plus the target file name might be updated for some reason but you can still define the same slug for the links' backward compatibility.